reference/wishlists.v3.yml

```diff
@@ -279,16 +292,12 @@ paths:
               type:
                 type: string
     put:
-      description: >-
-        Updates a Wishlist.
-
-
-        Use this endpoint to update an existing wishlist items, change the
-        wishlist name, customer ID or if the wishlist is available publicly.
+      description: |-
+        Updates a wishlist.
 
+        Use this endpoint to update existing wishlist items, change the wishlist's name and whether the wishlist is available publicly. To add or delete a wishlist item, see [Wishlist Items](https://developer.bigcommerce.com/api-reference/store-management/wishlists/wishlists-items).
 
         **Required Fields**
-
         * customer_id
       summary: Update a Wishlist
```

Comment:
```suggestion
      summary: Update a wishlist
```

Reply:
Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.